Makefile

```diff
@@ -71,12 +71,8 @@ help: ## Display this help.
 setup: ## Basic project setup, e.g. installing GitHook for checking license headers
 	cd .git/hooks && ln -fs ../../.githooks/* .
 
-
-##@ Development
-PYTHON3=python3
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: #controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(PYTHON3) ./hack/customize_crds.py
```

Comment:
Not sure, if we require this post-processing still somehow, if not, I would also delete the file hack/customize_crds.py.
But it is gone in operator_3.0 anyway

Reply:
thanks, delete the file too

Reply:
It's not used anywhere else so should be safe to remove it